### DISCUSSION TITLE
Help: Which code highlighting tool and theme was used in Next.js doc

### DISCUSSION THREAD

**Discussion by lonelychinaday** [vercel/next.js#64482] [Help] [answered]
### Summary

origin code: 
![Snipaste_2024-04-15_11-01-37](https://github.com/vercel/next.js/assets/42992672/67f512ff-e87b-4844-b677-d0f07b779efb)
appearance:
<img width="677" alt="image" src="https://github.com/vercel/next.js/assets/42992672/af0a7ff3-4cb6-458e-8fa7-379d9ac28fef">
Which code highlighting tool and theme was used in Next.js doc

**Comment by icyJoseph** [comment, marked as answer]
Hi,

I remember answering to this way back, and back then it was not so easy to figure out:

- https://github.com/vercel/next.js/discussions/37814

That being said, I did a bit of digging, and found that they are using: https://github.com/shikijs/shiki as syntax highlighter, with this CSS doing the last tweaks: https://nextjs.org/_next/static/css/8886293df12842d8.css ~ and, some CSS vars coming from, https://nextjs.org/_next/static/css/a490e627d440c40d.css

Which pointed towards `geist` design system by Vercel, found here: https://vercel.com/geist/code-block and there you have a CodeBlock `internal` component. 

The geist docs seem to be using Prism, with `undefined` theme, to syntax highlight though, but I guess, that's just an implementation detail.

So there ends our luck :)

**Reply by lonelychinaday** [reply]
Thank you very much for the reply, it was very helpful for me
